### ISSUE TITLE
Scaling after update

### ISSUE THREAD

**Issue by kuhnkeF** [1adrianb/face-alignment#41]
Hi,
thanks for sharing your code.

After your last update the new scale value is 195:
`scale = (d.right() - d.left() + d.bottom() - d.top()) / 195.0`
but 3D predictions are still scaled with 200:
`pts_img = torch.cat((pts_img, depth_pred * (1.0 / (256.0 / (200.0 * scale)))), 1)`

is this intended?

**Comment by 1adrianb** [comment]
Hi @kuhnkeF,

Yes, it was intended and it was done in order to bring it at close as possible to the lua torch version (see: https://github.com/1adrianb/2D-and-3D-face-alignment/blob/fc4d29999a1f0b8a833805bfe437bd32df886242/utils.lua#L187 )

This value is however dependent on the face detector used, so it can be probabably further be tweaked.

You should however notice an increase in the accuracy after the last update. If this is not the case, please let me know.

**Comment by kuhnkeF** [comment]
Thank you. 
I don't have any real comparison for the accuracy, but it seems to be better.
What are the best settings that I should use if the bounding boxes are created from ground truth landmarks?

**Comment by 1adrianb** [comment]
The existing values are already picked with the assumption that the bounding box is close to a tight bounding box (a ground truth one). I did play with values in the 190-220 range at some point but I didn't notice a major difference.

**Comment by kuhnkeF** [comment]
great! Thanks again. I think you can close this issue.
